constants.go

```diff
@@ -5,12 +5,13 @@ import "time"
 const (
 	pollingInterval time.Duration = 10 * time.Second
 
-	envVarRepoFullName     string = "GITHUB_REPOSITORY"
-	envVarRunID            string = "GITHUB_RUN_ID"
-	envVarRepoOwner        string = "GITHUB_REPOSITORY_OWNER"
-	envVarToken            string = "INPUT_SECRET"
-	envVarApprovers        string = "INPUT_APPROVERS"
-	envVarMinimumApprovals string = "INPUT_MINIMUM-APPROVALS"
+	envVarRepoFullName       string = "GITHUB_REPOSITORY"
+	envVarRunID              string = "GITHUB_RUN_ID"
+	envVarRepoOwner          string = "GITHUB_REPOSITORY_OWNER"
+	envVarToken              string = "INPUT_SECRET"
+	envVarApprovers          string = "INPUT_APPROVERS"
+	envVarMinimumApprovals   string = "INPUT_MINIMUM-APPROVALS"
+	envVarApprovalIssueTitle string = "INPUT_APPROVAL-ISSUE-TITLE"
```

Comment:
My assumption on this was that the hyphen was necessary because of the values in the `action.yaml`. Where it creates the input env var from a direct translation of the `minimum-approvals` in `action.yaml`. Maybe this is not the case?

Reply:
You are absolutely right, good catch.